adapters/src/main/java/io/realm/RealmBaseAdapter.java

```diff
@@ -61,6 +64,20 @@ public void onChange(BaseRealm results) {
         }
     }
 
+    public RealmBaseAdapter(@Nullable OrderedRealmCollection<T> data) {
+        this.adapterData = data;
```

Comment:
@cmelchior those lines are copy exactly from the other constructor >.< ! What do I do if data is not managed?